index.html

```diff
@@ -125,6 +125,7 @@ <h2>No Plugin Necessary</h2>
   <ul class="editor-logos">
     <li><a href="http://barebones.com/support/technotes/editorconfig.html"><img src="logos/bbedit.png" alt="BBEdit"><span>BBEdit</span></a></li>
     <li><a href="https://github.com/eranif/codelite/tree/master/EditorConfigPlugin"><img src="logos/codelite.png" alt="CodeLite"><span>CodeLite</span></a></li>
+    <li><a href="https://github.com/elementary/code"><img src="logos/elementary-code.png" alt="elementary Code"><span>elementary Code</span></a></li>
```

Comment:
Does it have an individual page? I couldn't find it but it seems like there should be one. If not, can you change the link to `https://github.com/elementary/code#readme`?

Reply:
@cassidyjames Does it have an individual page?
@xuhdev I changed the link. If it doesn't have a page, please merge it 🙇